includes/admin/class-sensei-setup-wizard.php

```diff
@@ -35,11 +35,14 @@ class Sensei_Setup_Wizard {
 		'features'  => [
 			'selected' => [],
 		],
+		'theme'     => [
+			'install_sensei_theme' => false,
+		],
 		'purpose'   => [
 			'selected' => [],
 			'other'    => '',
 		],
-		'__version' => '2',
+		'__version' => '3',
```

Comment:
Notice that it also resets the previous options of the users. But I don't think it's a big deal since it's the Setup Wizard choices that the users should go through a single time (or maybe not expect to be saved in case they return in the future).